### PR TITLE
Initial commit for OrganizationAvailabilitySet and OrganizationAvailability

### PR DESCRIPTION
**Pull request by wongvincent**
To be used by organizations: https://github.com/BrightspaceHypermediaComponents/organizations/pull/132

### CODE REVIEW

File: src/organizations/OrganizationAvailabilitySetEntity.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+import { Entity } from '../es6/Entity.js';
+import { Rels, Classes, Actions } from '../hypermedia-constants';
+import { OrganizationEntity } from './OrganizationEntity.js';
+
+/**
+ * OrganizationAvailabilitySetEntity class representation of a d2l OrgUnitAvailabilitySet.
+ */
+export class OrganizationAvailabilitySetEntity extends Entity {
+	_getCurrentOrgUnitAvailabilityHref() {
```
Comment (wongvincent) [inline src/organizations/OrganizationAvailabilitySetEntity.js]: this function is kind of gross, but i need a way to determine if there is an explicit entity in the OrganizationAvailabilitySet that matches the currentOrgUnit.

Comment (ctwomey1) [reply src/organizations/OrganizationAvailabilitySetEntity.js]: Please don't parse the orgunitid from the href. Every time we've done this it eventually caused us grief.

Let's talk Monday about alternative ways to do this

Comment (mdgbayly) [reply src/organizations/OrganizationAvailabilitySetEntity.js]: Yeh, would agree we don't want to be parsing URLs to get org unit ids.
Don't we have the href for the current org unit from the intial response?
So can't we pass it into these entity helpers from the UI?

Comment (mdgbayly) [reply src/organizations/OrganizationAvailabilitySetEntity.js]: Oh - you are using that... So isn't the problem that you need to use the `getOrganizationHref` against the set of entities that you load from the cache primer? i.e. compare the org unit href that you get from `getOrganizationHref` here, with the org unit href's in each of the `OrganizationAvailabilityEntity`s?

Is there something about the way the UI is being built that makes that hard?

**Pull request merged**